diff --git a/dist/index.js b/dist/index.js
index 36224d1dbd0616f7c375e3c21380780c7fbb8cc4..4b6261327117f2ba3d929d1854a347a9f24a9231 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -12412,12 +12412,10 @@ var matchSchPortsToSymbolPorts = ({
 // lib/utils/point-pairs-to-matrix.ts
 import { compose as compose12, scale as scale7, translate as translate12 } from "transformation-matrix";
 function pointPairsToMatrix(a1, a2, b1, b2) {
-  const tx = a2.x - a1.x;
-  const ty = a2.y - a1.y;
   const originalDistance = Math.sqrt((b1.x - a1.x) ** 2 + (b1.y - a1.y) ** 2);
   const transformedDistance = Math.sqrt((b2.x - a2.x) ** 2 + (b2.y - a2.y) ** 2);
   const a = transformedDistance / originalDistance;
-  const translateMatrix = translate12(tx, ty);
+  const translateMatrix = translate12(a2.x - a * a1.x, a2.y - a * a1.y);
   const scaleMatrix = scale7(a, a);
   return compose12(translateMatrix, scaleMatrix);
 }
